packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -96,6 +76,16 @@
 </script>
 
 <docs lang="mdx">
```

Comment:
This allows you to ignore the max-length inside mdx
```suggestion
<!--eslint-disable max-len -->
<docs lang="mdx">
```

Reply:
I tried but didn't work 😭 :  I checked this [discussion](https://github.com/mdx-js/mdx/discussions/2036) but I'm not sure what should be done in this situation.